spec/EmailVerificationToken.spec.js

```diff
@@ -359,7 +359,7 @@ describe("Email Verification Token Expiration: ", () => {
         followRedirect: false,
       }, (error, response) => {
         expect(response.statusCode).toEqual(302);
-        expect(response.body).toEqual('Found. Redirecting to http://localhost:8378/1/apps/invalid_link.html');
+        expect(response.body).toEqual('Found. Redirecting to http://localhost:8378/1/apps/verify_email_success.html?username=testEmailVerifyTokenValidity');
```

Comment:
I think it makes more sense to direct a user that page than the invalid_link or invalid_verification_link page. I made the change based on clicking on a few verification links in my inbox (that I had already verified) and more than half of them did showed the success page (small sample size though)